pkgs/applications/audio/rofi-mpd/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, python37Packages, fetchFromGitHub, pkgs }:
+
+with python37Packages; python37Packages.buildPythonPackage rec {
```

not the biggest fan of a top level `with` clause, but this is a little cleaner:
```suggestion
with python37Packages; 
buildPythonPackage rec {
```

---

if this isn't meant to be used by other python package, you should use buildPythonApplication as well :)